hms-app/src/components/IdeaCardFoldable.tsx

```diff
@@ -12,14 +12,38 @@ import {
   AvatarsGroup,
   useAccordionState,
 } from '@mantine/core'
-import { Idea } from '../common/types'
+import {
+  Category,
+  HackathonPreview,
+  IdeaPreviews,
+  Owner,
+  Participant,
+  Skill,
+} from '../common/types'
+import { getIdeaDetails } from '../actions/GetBackendData'
 
 type IProps = {
-  idea: Idea
+  ideaPreview: IdeaPreviews
   index: number
   type: string
 }
 
+type Idea = {
```

Comment:
you defined an idea type in your `types.tsx` file, why not use that here?

Reply:
I guess you also have the `error` and `loading` state flags, but you could do something like:
```
type IdeaCardState = {
  errorIdeaData: boolean
  isLoadingIdeaData: boolean
  idea: Idea
}
```